analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -134,6 +135,17 @@ class Gradle(
     // The path to the root project. In a single-project, just points to the project path.
     private lateinit var rootProjectDir: File
 
+    // Filter Gradle projects that are managed via Flutter / Pub. These projects are analyzed from within Pub.
+    override fun mapDefinitionFiles(definitionFiles: List<File>): List<File> {
+        val pubFactory = Pub.Factory()
+
+        return definitionFiles.filter { gradleDefinitionFile ->
+            pubFactory.globsForDefinitionFiles.none { pubDefinitionFile ->
+                gradleDefinitionFile.parentFile.searchUpwardsForFile(pubDefinitionFile) != null
+            }
+        }
+    }
```

Comment:
Just a note: it's not considering whether the package manager 'Flutter' is actually enabled.
IMO that's ok, just wanted to make sure it's considered.

Reply:
AFAIU that's because Flutter is not package manager, but a framework.